### PR TITLE
OHRM5X-1202: Add missing language strings in admin/pim/leave/time

### PR DESCRIPTION
**Pull request by SharukaNP**

### CODE REVIEW

File: symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue [hunk 1]
```diff
@@ -162,7 +162,7 @@
               <oxd-text tag="p" class="tls-hint">
                 {{
                   $t(
-                    'admin.optional_-_the_mail_server_requires_the_use_of_tls_security',
+                    'admin.optional_the_mail_server_requires_the_use_of_tls_security',
```
Comment (RajithaKumara) [inline symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue:165]: :ok_hand:

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: installer/upgrader/Migrations/V5/generalLangString.yaml [hunk 2]
```diff
@@ -741,5 +741,68 @@ langStrings:
     value: 'Other Email'
     unitId: other_email
   -
-    value: 'Submit'
-    unitId: submit
+    value: 'test'
+    unitId: test
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply installer/upgrader/Migrations/V5/generalLangString.yaml:745]: removed

File: installer/upgrader/Migrations/V5/generalLangString.yaml [hunk 3]
```diff
@@ -741,5 +741,68 @@ langStrings:
...
+  -
+    value: 'Date Range'
+    unitId: date_range
+  -
+    value: 'yyyy-mm-dd'
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply installer/upgrader/Migrations/V5/generalLangString.yaml:750]: when we start date format localization we could set to change this one string and it should change everywhere.

File: installer/upgrader/Migrations/V5/generalLangString.yaml [hunk 4]
```diff
@@ -741,5 +741,68 @@ langStrings:
...
+    value: 'Click here'
+    unitId: click_here
+  -
+    value: 'to download OrangeHRM OS 4.9.'
+    unitId: to_download_ohrm_4x
+  -
+    value: 'Bad Request'
+    unitId: bad_request
+  -
+    value: 'Accepts upto 1 mb'
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply installer/upgrader/Migrations/V5/generalLangString.yaml:807]: Didn't we discuss to keep this as a static and if any customization in validation to customize the string?

File: installer/upgrader/Migrations/V5/helpLangString.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    unitId: getting_started_with_orangehrm_message_1
+  -
+    value: 'The following information repositories are available to help you understand the application:'
+    unitId: getting_started_with_orangehrm_message_2
+  -
+    value: 'Should you experience any issues, please do not hesitate to contact us on'
+    unitId: contact_us_on_message
+  -
+    value: 'We will be delighted to help.'
+    unitId: we_will_be_delighted_to_help
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply installer/upgrader/Migrations/V5/helpLangString.yaml:19]: I thought on that but here right now the email is hard coded on to the page so sending as a parameter would be redundant right?

File: symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue [hunk 6]
```diff
@@ -200,7 +200,7 @@
 
         <oxd-form-actions>
           <required-text />
-          <oxd-button display-type="ghost" label="Reset" @click="onReset" />
+          <oxd-button display-type="ghost" :label="Reset" @click="onReset" />
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue:203]: Changed

File: symfony/client/src/orangehrmAttendancePlugin/components/RecordAttendance.vue [hunk 7]
```diff
@@ -50,7 +50,7 @@
             :rules="rules.date"
             :disabled="!isEditable"
             type="date"
-            placeholder="yyyy-mm-dd"
+            :placeholder="$t('general.date_format')"
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply symfony/client/src/orangehrmAttendancePlugin/components/RecordAttendance.vue]: I'll  explain during the meeting today on the process I thought for this

File: installer/upgrader/Migrations/V5/generalLangString.yaml [hunk 8]
```diff
@@ -741,5 +741,68 @@ langStrings:
...
+    value: 'Replace Current'
+    unitId: module_forbidden
+  -
+    value: "The page you're trying to access has restricted access"
+    unitId: module_access_restriction
+  -
+    value: 'Launching Soon'
+    unitId: launching_soon
+  -
+    value: 'This page is being developed for the OrangeHRM OS 5.0 Beta version.'
```
[the remark this answers is not in the document]
Comment (SharukaNP) [reply installer/upgrader/Migrations/V5/generalLangString.yaml:790]: Chnaged Based on Kavinda's recomendation.

File: installer/upgrader/Migrations/V5/helpLangString.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (SharukaNP) [reply installer/upgrader/Migrations/V5/helpLangString.yaml:19]: Ok so I did a bit of research on this [https://docs.google.com/document/d/1qcd-jykSQnjTsuLd8T--NyFjT2zMXncS55TTPwrxA5Y/edit#](url)

File: installer/dbscript/dbscript-2.sql [hunk 9]
```diff
@@ -4615,4 +4615,4 @@ SET @auth_admin_verify_data_group_id := (SELECT `id` FROM ohrm_data_group WHERE
 INSERT INTO ohrm_user_role_data_group (`can_read`, `can_create`, `can_update`, `can_delete`, `self`, `data_group_id`,`user_role_id`)
 VALUES (1, 0, 0, 0, 0, @auth_admin_verify_data_group_id, @admin_role_id);
 
-INSERT INTO `ohrm_i18n_group` (`name`, `title`) VALUES ('attendance', 'Attendance');
+INSERT INTO `ohrm_i18n_group` (`name`, `title`) VALUES ('attendance', 'Attendance'), ('help', 'Help'),('authentication', 'Authentication');
```
Comment (RajithaKumara) [inline installer/dbscript/dbscript-2.sql:4618]:
```suggestion
INSERT INTO `ohrm_i18n_group` (`name`, `title`) VALUES ('attendance', 'Attendance'), ('help', 'Help'),('auth', 'Authentication');
```
Shall we go as `auth` cz it short

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: installer/dbscript/dbscript-2.sql (hunk 9 above)
Comment (SharukaNP) [reply installer/dbscript/dbscript-2.sql]: changed authentication group to auth

File: symfony/client/src/core/pages/ModuleUnderDevelopment.vue [hunk 10]
```diff
@@ -31,17 +31,15 @@
...
         />
         <oxd-text tag="p" class="orangehrm-module-under-development-text">
-          This page is being developed for the OrangeHRM OS 5.0 Beta version.
-          However, you may try out all of the features by installing OrangeHRM
-          OS 4.9.
+          {{ $t('general.this_page_is_being_developed') }}
           <a
             target="_blank"
             class="orangehrm-module-under-development-link"
             href="https://sourceforge.net/projects/orangehrm/files/stable/4.9/"
```
Comment (RajithaKumara) [inline symfony/client/src/core/pages/ModuleUnderDevelopment.vue:38]: check this also

Comment (SharukaNP) [reply symfony/client/src/core/pages/ModuleUnderDevelopment.vue]: added link

File: installer/upgrader/Migrations/V5/TranslationTool.php [hunk 11]
```diff
@@ -38,10 +38,13 @@ class TranslationTool
      */
     public function up(string $groupName)
     {
-        $filename = 'installer/upgrader/Migrations/V5/messages.bg_BG.xml';
-        $langCode = 'bg_BG';
-        $this->readTranslations($filename, $langCode);
-        $this->addTranslations($langCode, $groupName);
+        //$filename = 'installer/upgrader/Migrations/V5/messages.bg_BG.xml';
```
Comment (SharukaNP) [inline installer/upgrader/Migrations/V5/TranslationTool.php]: Has been removed in the next commit

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**